### ISSUE TITLE
How to handle translation in dyte?

### ISSUE THREAD

**Issue by MarufHassanPietech** [dyte-io/flutter-samples#13]
Can you please tell me how to handle translation/internationalization in dyte UI Kit?
For eg I have the texts here `Polls` in English, I want to show it as `opiniepeiling` when Dutch language is selected in the app. 

![Screenshot_1721653301](https://github.com/user-attachments/assets/a474ca11-ad9a-4d8d-ae2f-d3ad07a6ddef)

**Comment by mohamed-ehab51** [comment]
I faced the same problem and my solution is : I changed those texts in DYTE's files , just follow the functions that shows the dialogue you want to translate and change the text , not the best solution but it was the only one available

**Comment by MarufHassanPietech** [comment]
@mohamed-ehab51 I don't think that is a good way to do it. I see that there is arbPath provided but its not translating the files when i provide my path.
```dart
DyteUIKitBuilder.build(
        uiKitInfo: uiKitInfo,
        skipSetupPage: true,
        arbPath: 'lib/l10n',
      );
```
